test/lit.site.cfg.in

```diff
@@ -76,6 +74,12 @@ if "@CMAKE_GENERATOR@" == "Xcode":
     config.environment['PATH'] = \
       os.path.pathsep.join((xcode_bin_dir, config.environment['PATH']))
 
+if "@LLDB_ENABLE@" == "TRUE":
+    config.available_features.add('lldb')
+    for root, dirs, files in os.walk("@LLDB_BUILD_DIR@"):
+        if root.endswith("site-packages"):
+            config.substitutions.append(('%lldb-python-path', root))
```

Comment:
I think, we want to keep `lit.site.cfg.in` as simple as possible.
Could you move this logic to `test/lit.cfg`?
And, we don't need `LLDB_ENABLE` flag. 
i.e:
```python
# lit.site.cfg.in
config.lldb_build_root = "@LLDB_BUILD_DIR@"

# lit.cfg
if config.lldb_build_root != "":
    config.available_features.add('lldb')
    # .. discovery
    config.substitutions.append(('%lldb-python-path', discovered_path))
```